addon/components/listing.js

```diff
@@ -6,12 +6,15 @@ import {
   generatorsForNode,
   triplesForGenerator,
   triplesForScope,
+  SHACL,
 } from '@lblod/submission-form-helpers';
 import { getSubFormsForNode } from '../utils/model-factory';
 import { next } from '@ember/runloop';
 import ListingList from '@lblod/ember-submission-form-fields/components/listing/list';
 import ListingTable from '@lblod/ember-submission-form-fields/components/listing/table';
 
+const ORDER = SHACL('order');
```

Comment:
I'm not sure this is the correct predicate. [`qb:order`](https://www.w3.org/TR/vocab-data-cube/#ref_qb_order) also seems to be a thing, and we use that for a [similar use-case in loket](https://github.com/lblod/frontend-loket/blob/0c5c628a4120502fd016fd468f2197b5e968866e/app/components/rdf-form-fields/climate-subsidy-costs-table.js#L168).

Reply:
A lookup of the definition i think it is ok to use this.